<?php

class MY_Exceptions extends CI_Exceptions {

    public function __construct() {
        parent::__construct();
        $this->CI = & get_instance();
    }

    public function show_404($page = '', $log_error = TRUE){
//        redirect('pagenotfound');
        echo 'here2324';
        exit;
        $CI = & get_instance();
        $CI->load->view('my_notfound_view');
        echo $CI->output->get_output();
        exit;
    }

}

?>